frontend/apps/app/components/CommonLayout/GlobalNav/GlobalNav.tsx

```diff
@@ -1,4 +1,4 @@
-import { LayoutGrid, Settings } from '@liam-hq/ui/src/icons'
+import { LayoutGrid, Settings } from '@liam-hq/ui'
```

Comment:
With the following modification, import is no longer possible via src.
https://github.com/liam-hq/liam/pull/2786/files#r2244866943